snapshots/snapshot_stg_orders.sql

```diff
@@ -0,0 +1,34 @@
+{% snapshot snapshot_dim_customers %}
+{#
+environment aware target_schema
```

Comment:
I'd suggest cleaning up the explanation in the comments. Initially I didn't understand that you were proposing the same config and just deciding between defining the variable in the SQL file vs defining the variable on a project-wide level.

Reply:
Good idea. Cleaned it up and made it more relevant to the current examples